azure/components/setup-utils.yml

```diff
@@ -3,7 +3,8 @@ steps:
       sudo apt-get install git -y
       git clone https://github.com/NHSDigital/api-management-utils.git utils
       cd utils
+      git checkout apm-1191-common-pipelines
```

Comment:
should we not be able to get rid of this completely now ?   given that we're checking out utils already ?

Reply:
Will track this issue in another ticket.